src/display/editor/editor.js

```diff
@@ -1140,40 +1142,61 @@ class AnnotationEditor {
     const signal = this._uiManager.combinedSignal(ac);
```

Comment:
How about reducing a bit of object creation below?
```suggestion
    const signal = this._uiManager.combinedSignal(ac);
    const opts = { capture: true, passive: false, signal };
```